docs/guides/1.2.0-upgrade-guide.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.2.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-120-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+---
+
+# Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+
+Here is a list of what's new in 1.2.0
+
+## New Features
+
```

Comment:
Just to be clear - there were no audit log capabilities before this release?

Given that, should there be a sentence ahead of the three bullets, announcing the capability?

Reply:
yes correct, this release introduces the capability in the terraform provider.

We supported audit logging via V3 APIs.